### PR TITLE
GTK+3 support for more plugins

### PR DESCRIPTION
**Pull request by sardemff7**
Geany requires GTK+ 2.24 minimum, so all the used APIs are there directly.

### CODE REVIEW

**Comment by sardemff7** [comment]
(This version requirement is there since Geany 1.25, and Geany Plugins requires Geany 1.26.)

File: geanylatex/src/geanylatex.c [hunk 1]
```diff
@@ -2128,8 +2128,8 @@ add_menu_to_menubar(void)
 	{
 		/* Build up menu for menubar */
 		menu_latex = gtk_menu_item_new_with_mnemonic(_("_LaTeX"));
-		gtk_menu_shell_insert(
-			menubar, menu_latex, g_list_length(menubar->children)-1);
+		gtk_menu_shell_append(
+			menubar, menu_latex);
```
Comment (b4n) [inline geanylatex/src/geanylatex.c]: Is this the same?  I think that was the same change that altered GeanyVC behavior, wasn't it?

File: geanylatex/src/geanylatex.c [hunk 2]
```diff
@@ -2278,8 +2278,8 @@ add_menu_to_menubar(void)
 	if (menu_bibtex == NULL)
 	{
 		menu_bibtex = gtk_menu_item_new_with_mnemonic(_("_BibTeX"));
-		gtk_menu_shell_insert(
-			menubar, menu_bibtex, g_list_length(menubar->children)-1);
+		gtk_menu_shell_append(
+			menubar, menu_bibtex);
```
Comment (b4n) [inline geanylatex/src/geanylatex.c]: idem

**Comment by b4n** [comment]
Not tested, but mostly LGTM

File: geanylatex/src/geanylatex.c (hunk 1 above)
Comment (frlan) [inline geanylatex/src/geanylatex.c]: It makes a difference yes. I think we should add the same logic as on GeanyVC here.

**Comment by sardemff7** [comment]
Changed to the same logic as GeanyVC.

**Pull request merged**